cloudigrade/account/tasks.py

```diff
@@ -279,6 +279,8 @@ def persist_inspection_cluster_results_task():
     messages = read_messages_from_queue(
         settings.HOUNDIGRADE_RESULTS_QUEUE_NAME,
         HOUNDIGRADE_MESSAGE_READ_LEN)
+    logger.info(_('{} read {} message(s) for processing'.format(
+        'persist_inspection_cluster_results_task', len(messages))))
```

Comment:
The parentheses here are incorrectly nested. I suspect you want it like:
```
logger.info(
   _('{} read {} message(s) for processing').format(
        'persist_inspection_cluster_results_task', len(messages)
    )
)
```

Reply:
I would also recommend adding positional arguments here, in case we ever need to internationalize this string and the order of variables can change.

Reply:
@ilyavw and @infinitewarp addressed both comments.